edge-util/test/Microsoft.Azure.Devices.Edge.Util.Test.Common/KubernetesAttribute.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Util.Test.Common
+{
+    using System;
+    using Xunit.Sdk;
+
+    [TraitDiscoverer("Microsoft.Azure.Devices.Edge.Util.Test.Common.KubernetesDiscoverer", "Microsoft.Azure.Devices.Edge.Util.Test.Common")]
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true)]
```

Comment:
I understood that you are following what is done in integration and stress attribute; but I would like to ask, do we need "AllowMultiple=true"?